functions.sh

```diff
@@ -181,13 +181,13 @@ eerrorn()
 #
 eerror()
 {
-	if yesno "${EINFO_QUIET}"; then
-		printf " $*\n" >/dev/stderr
+	if yesno "${EERROR_QUIET}"; then
+		return 0
 	else
 		if ! yesno "${RC_ENDCOL}" && [ "${LAST_E_CMD}" = "ebegin" ]; then
-			printf "\n"
+			printf "\n" >&2
```

Comment:
Yeah, libeinfo seems to do something entirely different, so I have no basis for how this should work.